packages/website/ts/components/top_bar.tsx

```diff
@@ -272,25 +274,21 @@ export class TopBar extends React.Component<TopBarProps, TopBarState> {
         );
     }
     private renderDocsMenu() {
-        if (!this.isViewing0xjsDocs() && !this.isViewingSmartContractsDocs()) {
+        if (!this.isViewing0xjsDocs() && !this.isViewingSmartContractsDocs() || _.isUndefined(this.props.menu)) {
```

Comment:
do we still need the first part of this check if we now have this.props.menu ?

Reply:
I think it's a stricter check. If is possible we have a future page with a menu passed in that is not a docs page.